docs/src/opt_cep.md

```diff
@@ -73,7 +73,7 @@ An overview is provided in the following table:
 | type of storage implementation                     | -                | `storage_type`                 | "none", "simple" or "seasonal"              | ::String       | "none"        |
 | allowing conversion (necessary for storage)        | -                | `conversion`            | `true` or `false`                               | ::Bool         | false         |
 | allowing demand                                    | -                | `demand`            | `true` or `false`                               | ::Bool         | true         |
-| allowing dispatchable generation                   | -                | `dispatchable_generation`            | `true` or `false`                               | ::Bool         | false         |
+| allowing dispatchable generation                   | -                | `dispatchable_generation`            | `true` or `false`                               | ::Bool         | true         |
```

Comment:
It seems to me that this should be true by default. Can you double check if that is the case?

Reply:
That is true.